constitution.md

```diff
@@ -275,6 +276,10 @@ the membership shall consider at a General Meeting whether that person's members
 
 20.4 A person may only participate in a General Meeting if they have been a member of the Society for at least fourteen (14) days at the time of a General Meeting.
 
+20.5 An Annual General Meeting cannot be closed until all business listed in Section 16.1 has been transacted.
+
+20.6 A General Meeting must be closed before the time stated for the conclusion of the General Meeting in the notice given for that General Meeting, or if no such time was stated in that notice, four (4) hours after the time stated for the commencement of that General Meeting in that notice. However, Section 20.5 takes priority over this clause.
```

Comment:
Does introducing 20.5 and 20.6 create an issue where technically if the room loses quorum (or worse everyone leaves) and the business required in 16.1 is not complete, the meeting is never closed?

Reply:
Not any more.